tools/circleci-upload-to-s3.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source tools/circleci-helpers.sh
+
+set -euo pipefail
+
+CT_REPORTS=$(ct_reports_dir)
+
+if [ ! -d "${CT_REPORTS}" ]; then
+    echo "Skip uploading, because $CT_REPORTS directory does not exist"
+    exit 0
+fi
+
+echo "Uploading test results to s3"
+echo $(circleci_s3_url ${CT_REPORTS})
+
+
+# XXX please, reduce number of files
+FILE_COUNT=$(find "${CT_REPORTS}" -type f | wc -l)
+echo "Uploading $FILE_COUNT files"
+ls $CT_REPORTS
+
+
+sudo pip install awscli --upgrade --user
+sudo aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
```

Comment:
I'm pretty sure we don't need `sudo` here, I don't see reason to use root to upload stuff to S3.

Reply:
I remember there were some issues with that, let me check if it works without sudo

Reply:
yeah works, fixed

Reply:
+1